print-dhcp6.c

```diff
@@ -784,6 +790,43 @@ dhcp6opt_print(netdissect_options *ndo,
 			ND_PRINT(")");
 			break;
 
+		case DH6OPT_BOOTFILE_URL:
+			if (optlen < 2) {
+				ND_PRINT(" ?)");
+				break;
+			}
+			tp = (const u_char *)(dh6o + 1);
+			ND_PRINT(" ");
+			nd_printjn(ndo, tp, optlen);
+			ND_PRINT(")");
+			break;
+
+		case DH6OPT_SZTP_REDIRECT:
+		case DH6OPT_USER_CLASS:
+			ND_PRINT(" ");
+			tp = (const u_char *)(dh6o + 1);
+			first_list_value = TRUE;
+			remainder_len = optlen;
+			while (remainder_len > 2) {
```

Comment:
With `>= 2` this loop would make the final empty URI (if present) visible in the output.